targets/TARGET_STM/TARGET_STM32F3/device/stm32f3xx_hal_comp_ex.h

```diff
@@ -2395,8 +2385,7 @@
     ||                                                         \
     (((INPUT) == COMP_NONINVERTINGINPUT_IO1)))
 
-#define IS_COMP_WINDOWMODE(WINDOWMODE) (((WINDOWMODE) == COMP_WINDOWMODE_DISABLE) || \
```

Comment:
Can we add a comment "Changed for MBED" ? Ideally adding the PR/commit number...